decred/decred/crypto/crypto.py

```diff
@@ -13,9 +13,9 @@
 from blake256.blake256 import blake_hash
 import nacl.secret
 
-from decred import DecredError
-from decred.util import encode
-from decred.util.encode import ByteArray, unblobCheck
+from decred.decred import DecredError
+from decred.decred.util import encode
+from decred.decred.util.encode import ByteArray, unblobCheck
```

Comment:
Can't do this. Typically this would indicate that you are importing the package in a way that it wasn't intended. How are you using the package that requires this change?

Reply:
When I run tinywallet, without this change I was getting an importerror(unknown location), so I figured since it is absolute import this would help. The only other fix I could figure was the insertion of my projectpath using sys.path.insert(). 
This was what I used to make tinywallet run on my machine, I would glad if you can proffer a better way.

Reply:
Are you using the CLI command `poetry run tinywallet` to run tinywallet?

Reply:
You'll need to **pip install poetry** and **poetry install** first.

Sorry this stuff isn't better documented already.

Reply:
I have implemented your suggestions, and I still got the error I dropped on tinydecred room

Reply:
Are you referring to the IDE error?

Reply:
Yes, it's actually not IDE related.

Reply:
Neither buck nor I are seeing this, and we use different flavors of linux I believe. I'm not great with poetry, but It looks like the virtual environment it uses isn't being set up properly.